modules/usage/module.php

```diff
@@ -252,7 +262,11 @@ public function on_before_delete_post( $post_id ) {
 	 * @return array
 	 */
 	public function add_tracking_data( $params ) {
-		$params['usages']['elements'] = get_option( self::OPTION_NAME );
+		$params['usages']['elements'] = get_option( self::ELEMENTS_OPTION_NAME );
+		$params['usages']['documents'] = $this->global_settings_page_usage
+			->create_from_global()
+			->get_collection()
+			->all();
```

Comment:
Why not `get_option( self::DOCUMENT_OPTION_NAME )`

Reply:
Since it has class that wrap it and handle it already, one source of truth...

Reply:
Handled.